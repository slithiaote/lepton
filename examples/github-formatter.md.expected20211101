* Statement of purpose
This file provides an example of Github-flavored Markdown output by Lepton. It also serves as a test of the corresponding code.

* Assets and requirements
No assets. This test shows off Ocaml and Python programs.

* Method


```sh
# This is a comment
echo "Hello world."
```

```text
Hello world.
```
