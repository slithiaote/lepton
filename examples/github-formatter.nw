* Statement of purpose
This file provides an example of Github-flavored Markdown output by Lepton. It also serves as a test of the corresponding code.

* Assets and requirements
No assets. This test shows off Ocaml and Python programs.

* Method

<<shell -exec shell -chunk sh>>=
# This is a comment
echo "Hello world."
@
