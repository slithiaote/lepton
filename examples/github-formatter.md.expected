# Statement of purpose
This file provides an example of Github-flavored Markdown output by Lepton. It also serves as a test of the corresponding code.

# Usage

The github-formatter.nw file should be processed with Lepton:
```sh
lepton -format_with github github-formatter.nw -o github-formatter.md
```

The result should be identical to the file github-formatter.md.expected.
This is checked by Gitlab CI in the test stage.



```sh
# This is a comment
echo "Hello world."
```

```text
Hello world.
```
